### PR TITLE
19 add first hacking opportunity collect all with a for loop

### PR DESCRIPTION
**Pull request by kavyab2006**

### CODE REVIEW

**Review by Dan-Quin — APPROVED** [review]

**Comment by MangoMagoCane** [comment]
There may be ways to substantially refactor `C1L1.tsx`, `ChapterOne.tsx`, and `LearnContainer.tsx` to reduce the amount of repeated code.

File: frontend/src/components/Learn Lessons/C1L1.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <h6 style={{ color: "lightgray", marginLeft: "5%" }}> Sample Output: </h6>
+      <button style={{ marginLeft: "10%" }}> Collect All </button>
+
+      <p>
+        {" "}
+        I have added a onclick attribute, which just tells the computer what to do when the button is clicked. Since we
+        are trying to make a button that allows the user to collect all bits, I have made onclick equal to collectAll()
+        which is a function. I have also changed the text on the button to Collect All, as that is what we want the
+        button to do. Now we have added functionality, which is just what happens when you click the button, to our
+        button. However, the computer still doesn’t know what to do as we do not have a collectAll() function made. So,
```
Comment (MangoMagoCane) [inline frontend/src/components/Learn Lessons/C1L1.tsx:44]: Is it a stylistic choice to use `’` this apostrophe instead of the regular `'`?

**Comment by MangoMagoCane** [comment]
It also appears the contents of the learn container are not scrollable.

File: frontend/src/components/Learn Lessons/C1L2.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+export default function C1L2({ children }) {
+  return (
+    <div className={`normalLesson ${open && "showing"}`} style={{ color: "white" }}>
+      <h3 style={{ color: "white", textAlign: "left", marginLeft: "-3%", marginRight: "-3%" }}>
+        {" "}
```
Comment (MangoMagoCane) [inline frontend/src/components/Learn Lessons/C1L2.tsx:5]: What is the purpose of all the single space jsx blocks `{" "}`? They don't end up rendering in the DOM.

**Review by MangoMagoCane — APPROVED** [review]

**Pull request merged**